src/js/Controllers/DisplayCapabilities.js

```diff
@@ -2213,11 +2213,11 @@ let capabilities = {
 		]
 	},
 	"COMMON": {
-		"audioPassThruCapabilities": {
+		"audioPassThruCapabilities": [{
```

Comment:
```suggestion
		"audioPassThruCapabilitiesList": [{
```
`audioPassThruCapabilities` will always be processed as an object according to the `HMI_API`